airbyte-integrations/connectors/destination-qdrant/metadata.yaml

```diff
@@ -4,10 +4,23 @@ data:
       enabled: true
     oss:
       enabled: true
+  allowedHosts:
+    hosts:
+      - "${indexing.url}"
+      - api.openai.com
+      - api.cohere.ai
+      - "${embedding.api_base}"
```

Comment:
Just for my own understanding, I'm curious how these dynamic refs are parsed. I see `embedding.api_base` matches the config setting name but I'm not sure about `indexing.url`.

Reply:
It's referencing this config value: https://github.com/airbytehq/airbyte/blob/97fdd834b701d384a349e4fc8eb43c94b561b5fb/airbyte-integrations/connectors/destination-qdrant/destination_qdrant/config.py#L30-L31